.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/spotify/helios
+    parallelism: 6
+    shell: /bin/bash --login
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+      CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+      MAVEN_OPTS: -Xmx128m
+
+    machine: true
+    
+    steps:
+    # Machine Setup
+    #   If you break your build into multiple jobs with workflows, you will probably want to do the parts of this that are relevant in each
+    # The following `checkout` command checks out your code to your working directory. In 1.0 this is done implicitly. In 2.0 you can choose where in the course of a job your code should be checked out.
+    - checkout
+
+    # 'See docs on artifact collection here https://circleci.com/docs/2.0/artifacts/'
+    - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
+
+    - run:
+        name: Install dependencies
+        working_directory: ~/spotify/helios
+        command: |
+         apt-get -y -qq update
+         apt-get -y -qq install jq
+         update-alternatives --set java /usr/lib/jvm/jdk1.8.0/bin/java; sudo update-alternatives --set javac /usr/lib/jvm/jdk1.8.0/bin/javac; echo -e "export JAVA_HOME=/usr/lib/jvm/jdk1.8.0" >> $BASH_ENV
```

Comment:
could be worthwhile to extract `jdk1.8.0` or `1.8.0` to a parameter, see an example of how that is done in https://github.com/spotify/scio/blob/master/.circleci/config.yml for `scala_212_version`

Reply:
done!